client/core/core.go

```diff
@@ -1166,7 +1165,7 @@ func (c *Core) verifyRegistrationFee(wallet *xcWallet, dc *dexConnection, coinID
 
 	regConfs, err := wallet.Confirmations(coinID)
 	if err != nil {
-		log.Errorf("Error getting confirmations for %s: %v", hex.EncodeToString(coinID), err)
+		log.Errorf("Error getting confirmations for %s: %v", coinIDString(assetID, coinID), err)
```

Comment:
https://github.com/decred/dcrdex/pull/468 removed this wallet.Confirmations check, and the assetID input arg for `verifyRegistrationFee`.